docs/content/development/build.md

```diff
@@ -22,7 +22,7 @@ mvn clean package
 This will compile the project and create the Druid binary distribution tar under
 `distribution/target/druid-VERSION-bin.tar.gz`.
 
-This will also create a tarball that contains `mysql-metadata-storage` extension under 
+After adding `-Pdist` option, this will also create a tarball that contains `mysql-metadata-storage` extension under 
```

Comment:
We need to add more details about this option. The document should say what this option means and how different `mvn clean install` and `mvn clean install -Pdist` are. Also, please update [this command](https://github.com/apache/incubator-druid/pull/6162/files#diff-15788e24b4299e22aff8aba2ea46281fR19) to `mvn clean install -Pdist`

Reply:
Okay, i will improve this.

Reply:
Done.